libqtile/backend/wayland/core.py

```diff
@@ -386,6 +386,12 @@ def _process_cursor_motion(self, time):
                         self.cursor.y - self._hovered_internal.y
                     )
                 else:
+                    if self._hovered_internal:
+                        self._hovered_internal.process_pointer_leave(
+                            self.cursor.x - self._hovered_internal.x,
```

Comment:
I wondered about this. Do you end up with a negative value if the mouse leaves via the left or top edge of the window?

Do the co-ordinates really matter here? We need to know the mouse left the window but does it matter where the mouse is at that point?

Reply:
You do get a negative value. I just tested on X11 and you see the same there with leave-notify events. This makes sense as it's the pointer coords at the event, and the event can't be raised before it leaves the window. As for whether they matter, that depends on what the window is doing, but they have the information there to work out what to do, if they care about where the pointer left the window.